moderntribe/square-one
# Block middleware: global color theme

## What does this do/fix?

- Adds a global color theme field via Block Middleware

## QA

See screencasts in the main [PR](https://github.com/moderntribe/square-one/pull/1014).

## TODO
- [ ] Revert composer.json changes and do an upgrade once https://github.com/moderntribe/square1-field-models/pull/6 is released.

## Tests

Does this have tests?

- [ ] No, this doesn't need tests because...
